mysite/notes/models.py

```diff
@@ -0,0 +1,12 @@
+from django.db import models
+from django.utils import timezone
+import datetime
+
+
+class MyNotes(models.Model):
+    title = models.CharField(max_length=60)
+    description = models.TextField(max_length=300)
+    note_created_date = models.DateField('Date published')
```

Comment:
1. Add `boolean` field for `is_read` defaults to False
2. Add `boolean` field `is_active` defaults to True

When you view any note, mark it `is_read = True`
When you delete any note, mark it `is_active = False`